airflow/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -43,6 +44,8 @@ class RedshiftDataOperator(BaseOperator):
     :param secret_arn: the name or ARN of the secret that enables db access
     :param statement_name: the name of the SQL statement
     :param with_event: indicates whether to send an event to EventBridge
+    :param enable_batch_execution: Enable executing multiple query.
+    :param sqls: List of sql statement. This is required when ``enable_batch_execution`` is ``true``
```

Comment:
I'm not so happy with having both `sql` and `sqls`
To me it seems like we also don't need `enable_batch_execution` as it can be deduced from the type user passed.
if string passed to `sql` then it's regular mode
if list of string passed to `sql` then it's batch mode.

WDYT?